lib/__tests__/cookieToAndFromJson.spec.ts

```diff
@@ -1,15 +1,20 @@
+import { describe, expect, it } from 'vitest'
 import { Cookie } from '../cookie/cookie.js'
 
 describe('Cookie.toJSON()', () => {
   it('should serialize a cookie to JSON', () => {
+    const start = new Date().toISOString()
     const cookie = Cookie.parse(
       'alpha=beta; Domain=example.com; Path=/foo; Expires=Tue, 19 Jan 2038 03:14:07 GMT; HttpOnly',
     )
+    const finish = new Date().toISOString()
     if (!cookie) {
       throw new Error('This should not be undefined')
     }
     expect(cookie.toJSON()).toEqual({
-      creation: new Date().toISOString(),
+      // Sometimes we roll over a millisecond, so we need to check both
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+      creation: expect.toBeOneOf([start, finish]),
```

Comment:
With fake timers this can be:

```suggestion
      creation: new Date().toISOString(),
```